src/instruments/src/ISIS/AutoBrightness.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { useSimVar } from '@instruments/common/simVars';
+import { useInteractionEvent } from '@instruments/common/hooks';
+
+const useInterval = (callback: () => void, delay: number | null) => {
```

Comment:
I think this is already available in `react-components`. If it isn't, let's move it to ``instruments/src/Common`.